src/main/resources/marytts/language/dsb/abbreviations.csv

```diff
@@ -0,0 +1,130 @@
+"ca.","circa"
```

Comment:
We need a header row here, otherwise we cannot load the CSV...

Reply:
ok, just added